### ISSUE TITLE
Pocket's name can overflow its box and screen in All Pockets view

### ISSUE THREAD

**Issue by leafcutterant** [darkwallet/darkwallet#218]
Similarly to #215, a name long enough can do this.

![clipboard01](https://cloud.githubusercontent.com/assets/7106231/5882983/a78748ba-a34e-11e4-8a0c-95282dc5a34a.png)

**Comment by veox** [comment]
One possible solution to this is linked above - that doesn't address #215, though.

**Comment by veox** [comment]
P.S. This has to be tested against two types of names:

```
nametoolongnametoolongnametoolongnametoolongnametoolong
nametoolong nametoolong nametoolong nametoolong nametoolong
```

**Comment by leafcutterant** [comment]
This fixes #215 in both scenarios - closing. Thank you, @veox!

**Comment by leafcutterant** [comment]
This fixes #218 in both scenarios - thanks @veox! I recommend pulling the solution.

(Sorry, accidentally closed issue at the firs time.
Edit 2: wrote wrong issue #.)

**Comment by veox** [comment]
> This fixes #215 in both scenarios 

You probably meant #218 (this, overflow in pocket box). Code relating to #215 hasn't been touched.

**Comment by leafcutterant** [comment]
Yes, sorry for the mistake - edited to correct.
